api/controllers/console/auth/login.py

```diff
@@ -30,11 +30,6 @@ def post(self):
         except services.errors.account.AccountLoginError:
             return {'code': 'unauthorized', 'message': 'Invalid email or password'}, 401
 
-        try:
```

Comment:
Since we already setup the account in `account_service` while creating the `tenant`, there is no need to do this anymore.